src/telemetry/connectionTelemetry.ts

```diff
@@ -110,15 +108,22 @@ export async function getConnectionTelemetryProperties(
     const username = connectionString.username ? 'DEFAULT' : 'NONE';
     const authStrategy = authMechanism ?? username;
 
+    const hostname = await getHostnameForConnection(
+      dataService.getConnectionString()
+    );
+
     const [instance, cloudInfo] = await Promise.all([
       dataService.instance(),
-      getCloudInfoFromDataService(dataService),
+      getCloudInfoFromDataService(hostname),
     ]);
+    const isAtlas = mongoDBBuildInfo.isAtlas(connectionString.toString());
+    const atlasHostId = isAtlas ? hostname : null;
 
     preparedProperties = {
       ...preparedProperties,
       auth_strategy: authStrategy,
-      is_atlas: mongoDBBuildInfo.isAtlas(connectionString.toString()),
+      is_atlas: isAtlas,
```

Comment:
wow interesting that we didn't have this one :)

Reply:
We did have. I just read it from `mongoDBBuildInfo.isAtlas()` higher in code now.

Reply:
right! 🤦